roles/avans/templates/student_project_backup.sh.j2

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+exit_needed=$(False)
+
+
+        exit_needed=$(True)
+
+        # go to directory needed to be backedup
+
+
+        # See the section "Passphrase notes" for more infos.
+        #where it will be stored
+        export BORG_REPO={{ hostvars[project.host].backup_user }}@{{ hostvars[project.host].backup_host}}:{{ hostvars[project.host].backup_location}}
+
+        export BORG_PASSPHRASE={{.vault_password}}
+
+
+        # initialse backup directory
+
+        set -e $BORG_REPO 
+
+        # some helpers and error handling:
+        info() { printf "\n%s %s\n\n" "$( date )" "$*" >&2; }
+        trap 'echo $( date ) Backup interrupted >&2; exit 2' INT TERM
+
+        {% for project in atgm_projects %}
+            {% if project.host == ansible_nodename %}
+        
+                #giving information which backup is generated
+                info "Starting backup {{ hostvars[project.host].project_directory }}/{{ project.year }}/{{ project.name }}"
+
+                # Backup the most important directories into an archive named after
+                # the machine this script is currently running on:
+
+                borg create \
+                    --verbose                                         \
+                    --list                                            \
+                    --stats                                           \
+                    --show-rc                                         \
+                    --compression lz4                                 \
+                    --exclude-caches                                  \
+
+                    #gives the backup name docs-when-of who
+                    ::docs-$(date "+%Y-%m-%dT%H-%M-%S"){{ project.year }}/{{ project.name }} \

```

Comment:
If you want to do it this way (running borg create multiple times), I think it's safer if you do something like this:

```suggestion
                    ::project-{{ project.year }}-{{ project.name }}-$(date "+%Y-%m-%dT%H-%M-%S") \

```

and then later in the prune, prune the same number of times, with a prefix of `project-{{ project.year }}-{{ project.name }}`